src/interactive/types.ts

```diff
@@ -20,7 +19,7 @@ export interface IAuthoredState {
   useAuthoredData: boolean;
   prompt: string;
   hint: string;
-  sensorUnit?: Unit;
+  sensorUnit?: string;
```

Comment:
I noticed you removed the type defined in `sensor-definitions.ts` 

```typescript
export type Unit = keyof typeof SensorDefinitions
```

Obviously `string` will provide less effective type checking, but you probably ran into something that was a pain in the neck that was preventing you from using the `Unit` type. I think that is fine  👍

Reply:
If the Unit type wasn't causing any trouble, it might be good to put it back in so we can catch errors early in the future.

Reply:
Hey Noah! So oddly even though the keys of the SensorDefinitions are all strings from what I can tell, the Unit type was giving string | number | undefined. I think we only want the sensorUnit to be a string which is why I changed it. But I might be missing something! Please let me know what you think.